src/kontrol/cli.py

```diff
@@ -110,3 +136,437 @@ def rpc_args(self) -> ArgumentParser:
+
+    get_model = command_parser.add_parser(
+        'get-model',
+        help='Display a model for a given node.',
+        parents=[
+            kontrol_cli_args.config_args,
+            kontrol_cli_args.foundry_test_args,
+            kontrol_cli_args.logging_args,
+            kontrol_cli_args.rpc_args,
+            kontrol_cli_args.bug_report_args,
+            kontrol_cli_args.smt_args,
+            kontrol_cli_args.foundry_args,
+        ],
+    )
+    get_model.add_argument(
+        '--node',
+        type=node_id_like,
+        dest='nodes',
+        default=[],
+        action='append',
+        help='List of nodes to display the models of.',
+    )
+    get_model.add_argument(
+        '--pending', dest='pending', default=False, action='store_true', help='Also display models of pending nodes'
+    )
+    get_model.add_argument(
+        '--failing', dest='failing', default=False, action='store_true', help='Also display models of failing nodes'
+    )
+
+    return parser
+
+
+def read_toml_args(parser: ArgumentParser, args: Namespace, cmd_args: list[str]) -> Namespace:
+    def canonicalize_option(long_opt: str) -> None:
+        if long_opt in ['ccopt', 'I', 'O0', 'O1', 'O2', 'O3']:
+            toml_commands.append('-' + long_opt)
+        elif long_opt == 'includes':
+            toml_commands.append('-I')
+        elif long_opt == 'optimization-level':
+            level = toml_profile_args[long_opt] if toml_profile_args[long_opt] >= 0 else 0
+            level = level if toml_profile_args[long_opt] <= 3 else 3
+            toml_profile_args[long_opt] = ''
+            toml_commands.append('-O' + str(level))
+        elif long_opt == 'counterexample-information':
+            toml_commands.append('--counterexample-information')
+            toml_commands.append('--failure-information')
+        else:
+            toml_commands.append('--' + long_opt)
+
+    def canonicalize_negative_logic_option(long_opt: str) -> None:
+        switching_options = ['emit-json', 'minimize', 'use-booster']
+        if long_opt in switching_options:
+            toml_commands.append('--no-' + long_opt)
+        elif long_opt[:4] == 'no-' and long_opt[3:] in switching_options:
+            toml_commands.append('--' + long_opt[3:])
+
+    def get_profile(toml_profile: dict[str, Any], profile_list: list[str]) -> dict[str, Any]:
+        if len(profile_list) == 0 or profile_list[0] not in toml_profile.keys():
+            return toml_profile
+        elif len(profile_list) == 1:
+            return toml_profile[profile_list[0]]
+        return get_profile(toml_profile[profile_list[0]], profile_list[1:])
+
+    toml_args = {}
+    if args.config_file.is_file():
+        with open(args.config_file, 'rb') as config_file:
+            try:
+                toml_args = tomli.load(config_file)
+            except tomli.TOMLDecodeError:
+                _LOGGER.error('Input config file is not in TOML format')
```

Comment:
Since we're not raising a `ValueError`, the execution will continue, right? Maybe, in this case, you could explicitly state in the error message that, since the file is not valid TOML, it's being ignored/the default or CLI-provided values are to be used.